doc/whats_new/v0.24.rst

```diff
@@ -195,6 +195,12 @@ Changelog
   usage in `fit`. :pr:`18334` by `Olivier Grisel`_ `Nicolas Hug`_, `Thomas
   Fan`_ and `Andreas Müller`_.
 
+- |Efficiency| Histogram initialization is now done in parallel in
+  :class:`ensemble.HistGradientBoostingRegressor` and
+  :class:`ensemble.HistGradientBoostingClassifier` which results in speed
+  improvement for problems that build a lot of nodes.
```

Comment:
Speed improvement only if parallelism is used. Maybe make that clearer.

Reply:
isn't this clear from the beginning "Histogram initialization is now done in parallel" ?

Reply:
Since parallelism is enabled by default (if you use the binary packages which 99% of our users do) I would say "speed improvement for problems that build a lot of nodes on multicore machines."